__tests__/app.test.js

```diff
@@ -93,6 +93,83 @@ describe('GET /api/articles/:article_id', () => {
       })
 })
 
+describe('PATCH /api/articles/:article_id', () => {
+  test("Return status 200 and returns the updated article with votes added to", () => {
+    return request(app)
+    .patch("/api/articles/1")
+    .send({inc_votes: 1})  
+    .expect(200)
+    .then(({body}) => {
+      const article = body.article;     
+      expect(article.votes).toBe(101)
+    })
+  })
+
+  test("Return status 200 and returns the updated article with votes deducted", () => {
+    return request(app)
+    .patch("/api/articles/1")
+    .send({inc_votes: -1})  
+    .expect(200)
+    .then(({body}) => {
+      const article = body.article;     
+      expect(article.votes).toBe(99)
+    })
+  })
+
+  test("Return status 200 and returns the updated article with votes = 0 when deducting greater than votes held", () => {
+    return request(app)
+    .patch("/api/articles/1")
+    .send({inc_votes: -101})  
+    .expect(200)
+    .then(({body}) => {
+      const article = body.article;     
+      expect(article.votes).toBe(0)
+    })
+  })
+
+  test("Return status 400 and an error message when invalid id passed", () => {
+    return request(app)
+    .patch("/api/articles/one")
+    .send({inc_votes: 1})
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe("400 Bad Request - Invalid ID")
+    })
+  })
+
+  test("return status 400 and an error message when an empty object is in the request", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({})
```

Comment:
- On an empty object you should expect a 200 status and for nothing to have changed (so expect the data to be unchanged). You could have a default value of inc_votes = 0 

- It would be good to check if inc_votes value is an integer.